backend/services/auth_service.py

```
@@ -128,6 +134,100 @@ def login_with_password(username: str, password: str):
+                WHERE user_id = %s AND used = FALSE
+                """,
+                (row["user_id"],),
+            )
+            cur.execute(
+                """
+                INSERT INTO public.password_reset_tokens (token, user_id, expires_at)
+                VALUES (%s, %s, %s)
+                """,
+                (token_hash, row["user_id"], expires_at),
+            )
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)